docs/quick_start.rst

```diff
@@ -1,152 +1,20 @@
 Quick Start
 ===========
 
-This guide will walk you through the functionality of IPFX starting with the simplest examples
-and build on itself towards increasing complexity of the scope of the analysis and the size of data to be analyzed.
+To run:
```

Comment:
describe in a bit more detail what are we running

Reply:
added, thanks